witchcraft/internal/middleware/request_test.go

```diff
@@ -250,6 +250,19 @@ func TestRequestMetricHandlerWithTags(t *testing.T) {
 	}
 }
 
+func TestStrictTransportSecurity(t *testing.T) {
+	wRouter := wrouter.New(
+		whttprouter.New(),
+		wrouter.RootRouterParamAddRequestHandlerMiddleware(middleware.NewStrictTransportSecurityHeader()),
+	)
+
+	w := httptest.NewRecorder()
+	req, err := http.NewRequest(http.MethodGet, "http://localhost/", nil)
+	require.NoError(t, err)
+	wRouter.ServeHTTP(w, req)
+	assert.Equal(t, w.Header().Get("Strict-Transport-Security"), "max-age=31536000")
```

Comment:
use `w.Result().Header["Strict-Transport-Security"]` so that you are inspecting the response headers.